src/containerapp/azext_containerapp/custom.py

```diff
@@ -2640,3 +2641,70 @@ def list_environment_telemetry_otlp(cmd,
 
     return containerapp_env_def
 
+
+def list_replica_containerappsjob(cmd, resource_group_name, name, execution=None):
+    if execution is None:
+        executions = ContainerAppsJobPreviewClient.get_executions(cmd=cmd, resource_group_name=resource_group_name, name=name)
+        execution = executions['value'][0]['name']
```

Comment:
Recommend use a more safe way to get the name. For example, check the execution is not none, check [0] exist, then get('name')

Reply:
If the user wants to get logs with job name they will 100% have an execution.